libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -153,7 +153,7 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
     // @Param: LOG
     // @DisplayName: ADS-B logging
     // @Description: 0: no logging, 1: log only special ID, 2:log all
-    // @Values: 0:no logging,1:log only special ID,2:log all
+    // @Values: 0:disable GPS out,1:GCS Failsafe log only special ID,2:log all
```

Comment:
Uhm.. you've updated the description to the wrong parameter :)

Reply:
Thanks for catching this; I have no idea how that happened.